integration_tests/runJest.js

```diff
@@ -26,7 +43,16 @@ function runJest(dir, args) {
 runJest.json = function(dir, args) {
   args = Array.prototype.concat(args || [], '--json');
   const result = runJest(dir, args);
-  result.json = JSON.parse((result.stdout || '').toString());
+  try {
+    result.json = JSON.parse((result.stdout || '').toString());
+  } catch (e) {
+    throw new Error(`
+      Can't parse JSON.
```

Comment:
I find it kind of nice to have it this way. You can see the error message right away among the rest of the output (like stack traces and whatever the crashing process throws up with)
Plus it's easier to write these messages in the actual code (vs. 'first line' + 'second line' + 'forth line', etc.)